esx_service/vmdk_ops.py

```diff
@@ -1786,11 +1824,16 @@ def handleVmciRequests(port):
     cartel = c_int32()
     vmci_grab_listening_socket(port)
 
-
     while True:
+        # Check the flag and the counter - exit the loop if flag is set and counter equals 0
+        if (stopEvent.isSet() and (opsCounter.value == 0)):
```

Comment:
Why check for value equal to zero, if stop event is set then this loop should exit and wait for the counter to get to zero and exit after that.

Reply:
Yes agree, will fix.